### PR TITLE
Tox

### PR DESCRIPTION
**Pull request by jakleh**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+skip_install = true
+deps = pre-commit
+commands =
+    pre-commit run black --all-files --verbose
+    # if you use the walrus operator on Python 3.8 disable the flake8 check
+    pre-commit run flake8 --all-files --verbose
+    pre-commit run trailing-whitespace --all-files --verbose
+    pre-commit run debug-statements --all-files --verbose
+    pre-commit run end-of-file-fixer --all-files --verbose
+
```
Comment (taylorfturner) [inline tox.ini:52]: new line EOF

File: tox.ini [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+[testenv:precom]
+skip_install = true
+deps = pre-commit
+commands =
+    pre-commit run black --all-files --verbose
+    # if you use the walrus operator on Python 3.8 disable the flake8 check
+    pre-commit run flake8 --all-files --verbose
+    pre-commit run trailing-whitespace --all-files --verbose
+    pre-commit run debug-statements --all-files --verbose
+    pre-commit run end-of-file-fixer --all-files --verbose
```
Comment (taylorfturner) [inline tox.ini:51]: need new line

**Review by taylorfturner — APPROVED** [review]

File: tox.ini [hunk 3]
```diff
@@ -0,0 +1,51 @@
+[tox]
+envlist = py36, py37, py38, py38, py310, docs, pypi-description, manifest, precom
```
Comment (taylorfturner) [inline tox.ini:2]:
```suggestion
envlist = py36, py37, py38, py39, py310, docs, pypi-description, manifest, precom
```

**Review by taylorfturner — CHANGES_REQUESTED** [review]

**Review by taylorfturner — APPROVED** [review]

**Comment by taylorfturner** [comment]
dropped 3.6 from `envlist` due to EOL and removed from NEP 29 dependencies. Tested locally with `tox -e py3*` and all ran successfully

**Review by ksneab7 — APPROVED** [review]

**Pull request merged**